src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+  const strings = sourceString.split(';');
+
+  strings.forEach((string) => {
+    const [property, value] = string.split(':');
+
+    if (property && value) {
+      result[property.trim()] = value.trim();
+    }
+  });
```

Comment:
The 'forEach' loop is correctly used here to iterate over the array of CSS declarations. However, be aware of any edge cases where the input might not be formatted as expected, such as missing colons or additional whitespace.